ant-node/src/replication.rs

```diff
@@ -319,6 +319,13 @@ impl Node {
             }
         }
 
+        if !to_replicate.is_empty() {
```

Comment:
will this be somehow to restrict ? 
it's normal that having 1 or 2 nodes not having a record, among its closest_7
maybe lift up the bar to having less than majority (i.e. <=3) holders among the closest_7